commit d9374323ccdad641ae8aa886cd87df22dad8b33c
Author: John Vandenberg <jayvdb@gmail.com>
Date:   Thu Aug 12 12:10:51 2021 +0800

    CMakeLists.txt: Fix Windows library install dir

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c5c91a3..434a9ca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -116,7 +116,13 @@ else ()
 endif()
 
 install(FILES ${HDRS} DESTINATION ${CMAKE_INSTALL_FULL_INCLUDEDIR}/swiftnav)
-install(TARGETS swiftnav DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR})
+install(
+  TARGETS swiftnav
+  EXPORT swiftnav-export
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
 
 # unit tests
 if(libswiftnav_BUILD_TESTS OR libswiftnav_BUILD_TEST_LIBS)
